wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```
@@ -89,7 +90,34 @@ public static function getSubscribedEvents() {
     ];
   }
 
-  /**
+  public static function assignVolunteerAsCampInitiator(string $op, string $objectName, $objectId, &$objectRef) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$objectId'. (undefined)

(UnusedFormalParameter)